src/main/kotlin/frc/robot/subsystems/shooter/ShooterIOSim.kt

```diff
@@ -74,12 +73,7 @@ class ShooterIOSim : ShooterIO {
         topMotor.update(Timer.getFPGATimestamp())
         bottomMotor.update(Timer.getFPGATimestamp())
 
-        topRollerInputs.velocity.mut_replace(
-            topMotor.velocity, Units.RotationsPerSecond
-        )
-
-        bottomRollerInputs.velocity.mut_replace(
-            bottomMotor.velocity, Units.RotationsPerSecond
-        )
+        topRollerInputs.velocity = Units.RotationsPerSecond.of(topMotor.velocity)
+        bottomRollerInputs.velocity = Units.RotationsPerSecond.of(bottomMotor.velocity)
```

Comment:
Why?

Reply:
This is of type `Angle`.

Reply:
It's not a `mutableMeasure` anymore.

Reply:
Yeah, but the type is used for IOReal & IOSim

Reply:
Hmm right. So you changed this because Phoenix now returns `Measure`s?

Reply:
Ok. What copies, tho? Mutable measures shouldn't create copies

Reply:
Not when you update their values, but in their usage we will need to convert them to regular measure's or make it take a mutable type. In the end it's just annoying.